docs/style_guide.adoc

```diff
@@ -0,0 +1,152 @@
+= Style Guide
+:toc: macro
+:toc-title:
+
+toc::[]
+
+From the GitHub Manual of Style, which this style guide inherits from:
```

Comment:
This section comes from [GitHub's open source guide](https://github.com/github/opensource.guide/blob/master/docs/styleguide.md), which in turn inherits from the GitHub Manual of Style, which I wasn't able to find on Google.

Could we cite this document and/or remove this section?